### PR TITLE
Initial development

### PR DESCRIPTION
**Pull request by RothAndrew**
## what
* Migrate project from GitLab

## why
* Open source woo! 🎉 

## references
n/a

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+//  type        = string
+//  description = "Rancher's endpoint will be '{subdomain_rancher}.{hosted_zone_domain_name}'. {subdomain_rancher} can be multi-layered e.g. 'rancher.foo.bar'"
+//}
+//
+//variable "rancher_letsencrypt_email" {
+//  type        = string
+//  description = "Email address to use for Rancher's LetsEncrypt certificate"
+//}
+//
+//variable "rancher_letsencrypt_environment" {
```
Comment (ac-miller) [inline variables.tf:124]: Are these commented variables needing to stay for now?

Comment (ac-miller) [inline variables.tf]: Are these commented variables needing to stay for now?

Comment (RothAndrew) [reply variables.tf]: Nope, I'll remove them

**Review by ac-miller — APPROVED** [review]

**Pull request merged**